components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-external-activity.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable indent */
+import '@brightspace-ui/core/components/button/button-subtle.js';
+import './d2l-activity-content-lti-link-jump-icon.js';
+import { css, html } from 'lit-element/lit-element.js';
+import { labelStyles } from '@brightspace-ui/core/components/typography/styles.js';
+import { LocalizeActivityEditorMixin } from '../../mixins/d2l-activity-editor-lang-mixin.js';
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin.js';
+
+class ActivityContentLTILinkExternalActivity extends SkeletonMixin(LocalizeActivityEditorMixin(RtlMixin(MobxLitElement))) {
+
+	static get properties() {
+		return {
+			entity: { type: Object },
+			showIsOpened: { type: Boolean },
+			skeleton: { type: Boolean },
+		};
+	}
+
+	static get styles() {
+		return [
+			super.styles,
+			css`
+				.d2l-external-activity-outer-frame {
+					flex-direction: column;
+				}
+				.d2l-content-link-external-activity {
+					align-items: center;
+					display: flex;
+					justify-content: space-between;
+				}
+				.d2l-external-activity-inner-frame {
+					display: flex;
+					margin-top: 18px;
+					padding-top: 18px;
+				}
+			`,
+			labelStyles
+		];
+	}
+
+	constructor() {
+		super();
+		this.skeleton = true;
+		this.showIsOpened = false;
+		this.activityWindowPopout = null;
+	}
+
+	render() {
+		if (this.entity) {
+			this.skeleton = false;
+		}
+		return html`
+			<div class="d2l-external-activity-outer-frame d2l-skeletize-container">
+				<div class="d2l-content-link-external-activity">
+					<label class="d2l-label-text d2l-skeletize">
+						${this.localize('content.externalActivity')}
+					</label>
+					<d2l-button-subtle
+						text="${this.localize('content.openInNewWindow')}"
+						icon="tier1:new-window"
+						@click="${this._openPopout}"
+						skeleton=${this.skeleton}
```

Comment:
This should probably be a [Boolean attribute](https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions)
```suggestion
						?skeleton=${this.skeleton}
```